Source/Core/DocumentHeader.cpp

```diff
@@ -65,4 +65,21 @@ void DocumentHeader::MergePaths(StringList& target, const StringList& source, co
 	}
 }
 
+void DocumentHeader::MergeReources(Vector<Resource>& target, const Vector<Resource>& source, const String& base_path)
+{
+	for (auto const& s : source)
```

Comment:
```suggestion
	for (const Resource& script : source)
```
Let's try to be explicit where we can.